apps/site/components/Common/Searchbox/ChatSources/index.module.css

```
@@ -0,0 +1,53 @@
+@reference "../../../../styles/index.css";
+
+.chatSources {
+  @apply mb-4
+    flex
+    flex-nowrap
+    items-center
+    gap-3
+    overflow-x-scroll
+    scroll-smooth;
+
+  -ms-overflow-style: none;
+  scrollbar-width: none;
```

These utility classes don’t exist in Tailwind yet, but they can be used like this;

```suggestion
    scroll-smooth
    [-ms-overflow-style:none]
    [scrollbar-width:none];
```